introduction.md

```diff
@@ -0,0 +1 @@
+I am Francisco, the new gakusei from Mexico, even being a Gaijin I will honor my Sensei.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.